protocol/bin/enabled-calls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "chat.1.local.bulkAddToConv": {"promise": true},
```

Comment:
nothing seems to happen when I click 'Add X user[s]'. Is the core side supposed to be plumbed through already?

Reply:
Yeah it should be plumbed through. I see the button spin and then on success the thread we were adding users to.